app/src/components/TimestampOrderBookScroller.tsx

```diff
@@ -183,23 +185,32 @@ class TimestampOrderBookScroller extends Component<Props> {
                                         this.middleReferenceItem = createRef();
                                     }
                                     return (
-                                        <Box
+                                        <CSSTransition
                                             key={listItem.price}
-                                            // @ts-ignore
-                                            ref={listItem.isMiddle ? this.middleReferenceItem : null}
-                                            className={classes.pricePoint}
+                                            timeout={ANIMATION_TIME}
+                                            classNames={{
+                                                enter: classes.levelEnter,
+                                                enterActive: classes.levelEnterActive,
+                                                exit: classes.levelExit,
+                                                exitActive: classes.levelExitActive,
+                                            }}
```

Comment:
`classNames` (and not `className`) was possible ?

Reply:
`classNames` is actually a special prop for `CSSTransition` that specifies the css class the element will have for each transition phase (`enter`, `enterActive`, `exitActive`, `exit`). Can't be replaced by the basic `className`.